lib/doorkeeper/config.rb

```diff
@@ -237,6 +236,18 @@ def token_grant_types
       @token_grant_types ||= calculate_token_grant_types
     end
 
+    def refresh_token_revoked_on_use?
+      unless @refresh_token_revoked_on_use.nil?
+        return @refresh_token_revoked_on_use
+      end
+
+      @refresh_token_revoked_on_use =
+        ActiveRecord::Base.connection.column_exists?(
+          :oauth_access_tokens,
+          :previous_refresh_token
+        )
```

Comment:
@tute Unfortunately ||= will keep recomputing if the returned value is `false`
Thats why I have it explicitly for `nil?`